### ISSUE TITLE
In Terraform 0.11.8, aws_codebuild_project resource does not expose logsConfig from AWS API

### ISSUE THREAD

**Issue by emmanuel** [terraform-providers/terraform-provider-aws#6312]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

In the current implementation, the `aws_codebuild_project` resource does not expose the `logsConfig` from the upstream AWS SDK. The impact is that CodeBuild creates a log group with unbounded retention and the user cannot configure it.

### New or Affected Resource(s)

* aws_codebuild_project

### Potential Terraform Configuration

Proposed HCL:

```hcl
resource "aws_codebuild_project" "example" {
  name          = "test-project"
  description   = "test_codebuild_project"
  build_timeout = "5"
  service_role  = "${aws_iam_role.example.arn}"

  logs_config {
    cloudwatch_logs {
      status = "ENABLED|DISABLED"
      group_name = "..."
      stream_name = "..."
    }
    s3_logs {
      status = "ENABLED|DISABLED"
      location = "..." # ARN of an S3 bucket and the path prefix for S3 logs
    }
  }

  artifacts {
    ...
  }

  cache {
    ...
  }

  environment {
    ...
  }

  source {
    ...
  }

  vpc_config {
    ...
  }

  tags {
    ...
  }
}
```

### References

It appears that this option was simply omitted from the exposed interface. It is an optional field with
(somewhat) reasonable defaults, so this is very understandable.

There was an earlier pull request that exposed `vpc_config`, which is another optional member of
`aws_codebuild_project`.

I would be happy to submit an implementation. I would leverage as much as I could from #3324, which seems to be a very similar situation.

* https://docs.aws.amazon.com/sdk-for-go/api/service/codebuild/#CreateProjectInput
* #3324

**Comment by vibhuyadav** [comment]
I was about to open the same issue. Thanks @emmanuel

**Comment by bflad** [comment]
Support for managing the logging configuration has been merged and will release with version 2.19.0 of the Terraform AWS Provider, likely in the next two days. 👍 Thanks to @srhaber!

**Comment by bflad** [comment]
This has been released in [version 2.19.0 of the Terraform AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#2190-july-11-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.

For further feature requests or bug reports with this functionality, please create a [new GitHub issue](https://github.com/terraform-providers/terraform-provider-aws/issues/new/choose) following the template for triage. Thanks!
